content-src/asrouter/components/Button/_Button.scss

```diff
@@ -41,7 +41,7 @@
   }
 }
 
-[lwt-newtab-brighttext] {
+[lwt-newtab-brighttext]:not(.unthemed) {
```

Comment:
I would change the name to something more descriptive like `force-light-theme` or something similar.